ubuntu-advantage

```diff
@@ -86,12 +87,15 @@ Commands:
  disable-livepatch [-r]            disable the Livepatch service. With "-r", the
                                    canonical-livepatch snap will also be removed
  enable-cc-provisioning <TOKEN>    enable the commoncriteria PPA repository and
-                                   install the ubuntu-commoncriteria .DEB package
- disable-cc-provisioning           currently not supported
- enable-cisaudit                   enable the security benchmarks PPA repository
-                                   and install the ubuntu-cisbenchmark-16.04 DEB package.
+                                   install the ubuntu-commoncriteria DEB package
+ disable-cc-provisioning           disable the commoncriteria PPA repository and
+                                   remove the ubuntu-commoncriteria DEB package
+ enable-cisaudit <TOKEN>           enable the security benchmarks PPA repository
```

Comment:
I think you need to rebase on top of master, because the CIS audit tool is there already.

Reply:
Oh. Never mind :)